x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -73,7 +73,7 @@ export const FormContext: React.FC<Props> = ({
 
   const submitCase = useCallback(
     async (
-      { connectorId: dataConnectorId, fields, syncAlerts, ...dataWithoutConnectorId },
+      { connectorId: dataConnectorId, fields, syncAlerts = true, ...dataWithoutConnectorId },
```

Comment:
Why is this defaulted to `true`? I guess you are trying to have a default here in case the `disableAlerts = true` and we don't show the toggle. Is that case the `syncAlerts` should be false. Also here `x-pack/plugins/cases/public/components/create/schema.tsx` the field has a default value to `true` so I think it is no needed to be declared here. 

Either way, I think we should reflect hiding of the alert functionality to `syncAlerts = false`

Reply:
if alerting isnt on, does it matter? if it is on, dont we want it enabled?